packages/app-toolbox/src/Verify.tsx

```diff
@@ -206,4 +208,17 @@ class Verify extends React.PureComponent<Props, State> {
   }
 }
 
+const AlignedIcon = styled(Icon)`
+      &&&.ui--AlignedIcon::before {
+        position: relative;
+        left: 0.75rem;
+        top: 1rem; 
+        width: 32px;
+        height: 32px;
+        font-size: 24pt;
```

Comment:
Can we stick to `32px` here (i.e. no need to convert-in-the-head between the 2 measurements)

Reply:
fixed